odiglet/pkg/kube/instrumentation_ebpf/pods.go

```diff
@@ -106,20 +106,16 @@ func (p *PodsReconciler) instrumentWithEbpf(ctx context.Context, pod *corev1.Pod
 	return instrumentPodWithEbpf(ctx, pod, p.Directors, runtimeDetails, podWorkload)
 }
 
-func (p *PodsReconciler) getPodWorkloadObject(ctx context.Context, pod *corev1.Pod) (*common.PodWorkload, error) {
+func (p *PodsReconciler) getPodWorkloadObject(ctx context.Context, pod *corev1.Pod) (*workload.PodWorkload, error) {
 	for _, owner := range pod.OwnerReferences {
-		name, kind, err := kubeutils.GetWorkloadNameFromOwnerReference(owner)
+		workloadName, workloadKind, err := workload.GetWorkloadFromOwnerReference(owner)
 		if err != nil {
-			if kubeutils.IsErrorKindNotSupported(err) {
-				return nil, nil
-			}
-
-			return nil, err
+			return nil, workload.IgnoreErrorKindNotSupported(err)
```

Comment:
We used to ignore (not return the error) the ErrorKindNotSupported. Is this change in behavior intentional?